src/Components/ServiceScene/JSONPanel/LogsJsonComponent.tsx

```diff
@@ -62,6 +64,12 @@ export default function LogsJsonComponent({ model }: SceneComponentProps<LogsJso
   const jsonParserPropsMap = new Map<string, AdHocFilterWithLabels>();
   const lineFilterVar = getLineFiltersVariable(model);
 
+  const idField = rawFrame?.fields.find((field) => isLogsIdField(field.name));
+  const lineIndex = idField?.values.findIndex((v) => v === selectedLine?.id);
+  const cleanLineIndex = lineIndex !== undefined && lineIndex !== -1 ? lineIndex : undefined;
+  const scrollToPath: ScrollToPath | undefined =
+    cleanLineIndex !== undefined ? [cleanLineIndex, JsonVizRootName] : undefined;
```

Comment:
```suggestion
  const scrollToPath: ScrollToPath | undefined = useMemo(() => {
    if (selectedLine === undefined) {
      return undefined;
    }
    const idField = rawFrame?.fields.find((field) => isLogsIdField(field.name));
    const lineIndex = idField?.values.findIndex((v) => v === selectedLine?.id);
    const cleanLineIndex = lineIndex !== undefined && lineIndex !== -1 ? lineIndex : undefined;
    return cleanLineIndex !== undefined ? [cleanLineIndex, JsonVizRootName] : undefined;
  }, [selectedLine]);
```

If you do this you would do less work on every render and skip NOOPs when there is no selectedLine